app/src/main/java/org/mozilla/focus/tiles/HomeTilesManager.kt

```diff
@@ -78,7 +81,12 @@ class BundledTilesManager private constructor(context: Context) {
     fun unpinSite(context: Context, uri: Uri): Boolean {
         for (pair in bundledTilesCache) {
             if (compareUri(uri, pair.key)) {
-                // TODO: Remove by pair.value.id (BEFORE removing from cache)
+                val blacklist = context.getSharedPreferences(PREF_HOME_TILES, MODE_PRIVATE).getStringSet(BUNDLED_SITES_BLACKLIST, mutableSetOf())
```

Comment:
Since you're using this in multiple places, just make a private fun to load the blacklist.